README.md

```diff
@@ -251,7 +251,7 @@ default configuration values (and structure of the configuration table) are:
 {
   ensure_dependencies   = true, -- Should packer install plugin dependencies?
   package_root   = util.join_paths(vim.fn.stdpath('data'), 'site', 'pack'),
-  compile_path = util.join_paths(vim.fn.stdpath('config'), 'plugin', 'packer_compiled.vim'),
+  compile_path = util.join_paths(vim.fn.stdpath('config'), 'plugin', 'packer_compiled.vim'), -- can also be a ".lua" file
```

Comment:
This should be updated to the new default of `packer_compiled.lua`

Reply:
Oops, good catch!